samples/graph-membership-change-notification/nodejs/README.md

```diff
@@ -0,0 +1,188 @@
+---
+page_type: sample
+description: This sample application demonstrates how to send notifications for shared channel events in Microsoft Teams, such as users being added, removed, or having their membership updated, and when channel is shared/unshared with a team using Node.js and the Microsoft Graph API.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- Nodejs
+extensions: 
+ contentType: samples
+ createdDate: "10-03-2022 11:30:15 AM"
+urlFragment: officedev-microsoft-teams-samples-graph-membership-change-notification-nodejs
+
+---
+
+# Change Notifications For Team and Channel Using Microsoft Graph Node.js
+
+This sample application demonstrates how to send notifications for shared channel events in Microsoft Teams, such as users being added, removed, or having their membership updated and when channel is shared/unshared with a team. The application leverages Node.js and the Microsoft Graph API to deliver real-time notifications. It includes comprehensive setup instructions covering Azure AD registration, bot configuration, self-signed certificate management, and deployment using the Microsoft 365 Agents Toolkit for Visual Studio Code.
+
+## Included Features
+* Tabs
+* Graph API
+* RSC Permissions
```

Comment:
you are mentioning RSC permission here, but below only classic permissions are added. You can probably include another section for RSC apps. Provide a sample manifest for RSC based on newer properties.